Makefile.am

```diff
@@ -152,6 +152,7 @@ myhtopheaders = \
 # -----
 
 linux_platform_headers = \
+	FileDescriptorMeter.h \
```

Comment:
I think this should be under `myhtopsources` to be available to the rest of the Platforms?

Reply:
It should suffice to mention this on each platform that want's to use that meter IMHO.